LiveStream/Models/LiveStreamEvent.swift

```diff
@@ -1,60 +1,68 @@
+// swiftlint:disable type_name
 import Argo
 import Curry
+import Prelude
 import Runes
 
 public struct LiveStreamEvent: Equatable {
-  public let creator: Creator
```

Comment:
here are all the changes i made to the live stream model.

* i flattened the stream sub-struct
* i made all `let`s into `public fileprivate(set) var`s
* i made some things optional cause in one api endpoint it comes back and in another it doesnt